datafusion/core/src/physical_plan/memory.rs

```diff
@@ -243,547 +238,3 @@ impl RecordBatchStream for MemoryStream {
-                "    ProjectionExec: expr=[a1@0 as a1, a2@1 as a2]",
-                "      SortPreservingMergeExec: [c1@2 ASC NULLS LAST]",
-                "        ProjectionExec: expr=[SUM(aggregate_test_100.c4) PARTITION BY [aggregate_test_100.c1] ORDER BY [aggregate_test_100.c9 ASC NULLS LAST] ROWS BETWEEN 1 PRECEDING AND 1 FOLLOWING@3 as a1, COUNT(UInt8(1)) PARTITION BY [aggregate_test_100.c1] ORDER BY [aggregate_test_100.c9 ASC NULLS LAST] ROWS BETWEEN 1 PRECEDING AND 1 FOLLOWING@4 as a2, c1@0 as c1]",
-                "          BoundedWindowAggExec: wdw=[SUM(aggregate_test_100.c4): Ok(Field { name: \"SUM(aggregate_test_100.c4)\", data_type: Int64, nullable: true, dict_id: 0, dict_is_ordered: false, metadata: {} }), frame: WindowFrame { units: Rows, start_bound: Preceding(UInt64(1)), end_bound: Following(UInt64(1)) }, COUNT(UInt8(1)): Ok(Field { name: \"COUNT(UInt8(1))\", data_type: Int64, nullable: true, dict_id: 0, dict_is_ordered: false, metadata: {} }), frame: WindowFrame { units: Rows, start_bound: Preceding(UInt64(1)), end_bound: Following(UInt64(1)) }], mode=[Sorted]",
-                "            SortExec: expr=[c1@0 ASC NULLS LAST,c9@2 ASC NULLS LAST]",
-                "              CoalesceBatchesExec: target_batch_size=8192",
-                "                RepartitionExec: partitioning=Hash([Column { name: \"c1\", index: 0 }], 8), input_partitions=8",
-                "                  RepartitionExec: partitioning=RoundRobinBatch(8), input_partitions=1",
-            ]
-        };
-
-        let actual: Vec<&str> = formatted.trim().lines().collect();
-        let actual_len = actual.len();
-        let actual_trim_last = &actual[..actual_len - 1];
-        assert_eq!(
-            expected, actual_trim_last,
-            "\n\nexpected:\n\n{expected:#?}\nactual:\n\n{actual:#?}\n\n"
-        );
-        Ok(())
-    }
-
-    // DummyPartition is a simple implementation of the PartitionStream trait.
-    // It produces a stream of record batches with a fixed schema and the same content.
-    struct DummyPartition {
-        schema: SchemaRef,
-        batch: RecordBatch,
-        num_batches: usize,
-    }
-
-    impl PartitionStream for DummyPartition {
-        // Return a reference to the schema of this partition.
-        fn schema(&self) -> &SchemaRef {
-            &self.schema
-        }
-
-        // Execute the partition stream, producing a stream of record batches.
-        fn execute(&self, _ctx: Arc<TaskContext>) -> SendableRecordBatchStream {
-            let batches = itertools::repeat_n(self.batch.clone(), self.num_batches);
-            Box::pin(RecordBatchStreamAdapter::new(
-                self.schema.clone(),
-                futures::stream::iter(batches).map(Ok),
-            ))
-        }
-    }
-
-    // Test the less-lock mode by inserting a large number of batches into a table.
-    #[tokio::test]
-    async fn test_one_to_one_mode() -> Result<()> {
```

Comment:
Given there are no locking differences now between different partitioning combinations of insert to execute, I don't think these tests are adding any additional coverage that is not in 

https://github.com/apache/arrow-datafusion/blob/eb918ab217213d5e07e71e53c118a8409d2f71a0/datafusion/core/src/datasource/memory.rs#L455-L476

So I opted to remove them